lib/usage.js

```diff
@@ -294,6 +295,9 @@ module.exports = function usage (yargs, y18n) {
         if (~options.number.indexOf(key)) type = `[${__('number')}]`
 
         const extra = [
+          (key in deprecatedOptions)
+            ? `[${__('deprecated')}${typeof deprecatedOptions[key] === 'string'
```

Comment:
Could you please add the new translation(s) to `locales/en.json`?

Reply:
done